shell/platform/windows/external_texture_gl.cc

```diff
@@ -0,0 +1,126 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/windows/external_texture_gl.h"
+
+// OpenGL ES and EGL includes
+#include <EGL/egl.h>
+#include <EGL/eglext.h>
+#include <EGL/eglplatform.h>
+#include <GLES2/gl2.h>
+#include <GLES2/gl2ext.h>
+
+#include <iostream>
+
+namespace flutter {
+
+typedef void (*glGenTexturesProc)(GLsizei n, GLuint* textures);
```

Comment:
There should be an anonymous namespace her containing everything that doesn't correspond to a declaration in external_texture_gl.h